### ISSUE TITLE
Localization for Mexico

### ISSUE THREAD

**Issue by marvelousbeat** [flectrahq/discussion#61]
Are there any plans to include the local accounting module that was developed for Odoo 11? If there are plans how can I support the development?

**Comment by agb80** [comment]
@marvelousbeat at @openpymemx we have being working on a full odoo distribution that currently supports CFDI 3.3 and eAccounting 1.3. Also we actively look for partners for work together. Here you can see some of the work we have done: https://www.youtube.com/channel/UCVH4PLttdt09h1Di4yA2aUA

Look specially to this video: https://www.youtube.com/watch?v=TmyBHrwRdy4

**Comment by parthivgls** [comment]
@agb80  is it open source and compatible with odoo 11 ?

**Comment by agb80** [comment]
@parthivgls yes it is open source and fully developed for us. Currently there is no public repo because the strategy we are using to fund development is pay a suscription for get access to the code and support.

Regarding compatibility, we have being using odoo 8 as base for our work, do not have try it on odoo 11 yet. The electronic invoicing is entirely build on a custom addon with almost no dependency to any odoo module, so I think could be easily adapted to odoo 11. 

What could worried me is the eAccounting part. eAccounting is a special requirement from Mexican authorities that enforces all business to send a trial balance, chart of accounts and journal items every month using a special XML template for each report. At this moment we are tied to how odoo 8 accounting module works but could be a very good opportunity to migrate to new odoo versions.

**Comment by marvelousbeat** [comment]
@agb80 That's great I'm very glad there are some bases we could work with, we could try to port your version to odoo 11 which is also compatible with Flectra, would you be willing to give it a try?

**Comment by agb80** [comment]
Sure, we can give you access to our prive repos and you can start working on that. Send me a mail to discuss further

**Comment by marvelousbeat** [comment]
@agb80 I sent you and email, please let me know if you got it so we can arrange how to work on the project.
